modules/SystemsBiology.py

```diff
@@ -0,0 +1,164 @@
+from settings import Config
+from common.HGNCParser import GeneParser
+import sys
+import logging
+import datetime
+import opentargets.model.core as opentargets
+import opentargets.model.bioentity as bioentity
+import opentargets.model.evidence.core as evidence_core
+import opentargets.model.evidence.association_score as association_score
+
+__copyright__ = "Copyright 2014-2018, Open Targets"
```

Comment:
this `__*__` has to disappear in favor of having a LICENSE file in the root folder and also adding to the README.md a [copyright like this](https://github.com/opentargets/data_pipeline/blob/master/README.md#copyright)

Reply:
This line is present in all these modules, so it probably makes sense to tidy them all up in one go.